lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
+        platform_family 'cisco-wrlinux'
```

Comment:
That rationale seems entirely wrong.  The platform_family is designed to cover different operating systems that come from a common base where the distros DO make different assumptions.  For example both "ubuntu" and "debian" have a platform_family of "debian".

If we start going down the road of requiring platform_family values to all make identical assumptions, then it dilutes the utility of platform_family down to only being useful for 'rhel' which makes the entire concept useless.